src/m4/jtr_x86_logic.m4

```diff
@@ -464,16 +464,17 @@ dnl ======================================================================
 
   AS_IF([test "x$CPU_NOTFOUND" = x0],
   [
-  AC_MSG_CHECKING([for AVX512BW])
+  AC_MSG_CHECKING([for AVX512BW + AVX512VL + AVX512DQ])
```

Comment:
... and here.

(I don't get why we have this in two places.)

Reply:
> I don't get why we have this in two places.

The first half of `m4/jtr_x86_logic.m4` checks what the build host supports (using `cpuid`), unless cross compiling. The second half of it is (only) for cross compiling, so it just checks what the toolchain can do.